src/components/Calculator.js

```diff
@@ -1,17 +1,36 @@
-import React from 'react';
```

Comment:
[OPTIONAL] On large screens, the calculator appears way down the screen. I don't think this should be the expected behavior. The calculator could probably be better being static around the middle of the window. You might want to give a look at this.

![image](https://user-images.githubusercontent.com/79114233/143477314-0ab0acd2-4743-480f-ace7-b22f97b8da61.png)